src/tmerc.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * @module ngeo.proj.tmerc
+ */
```

Comment:
This comment is not needed.

Reply:
@gberaudo I did as in https://github.com/geoblocks/proj/blob/master/src/somerc.js#L2
But will remove it if you think it is not relevant.

Reply:
It is especially the ngeo prefix which makes no sense here.
You can keep the `@module`, it does not hurt but I think it is only needed by jsdoc, which we do not use.

Reply:
If you decide to remove completely then it would be great to remove usage in the other places.